service/src/main/java/bio/terra/user/app/configuration/SamConfiguration.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.user.app.configuration;
```

Comment:
My understanding is that since it'll be the UI and other front end services talking to this, SAM is needed to auth the user and to grab their persistent ID for use in the db.